test/spec/modules/rubiconAnalyticsSchema.json

```diff
@@ -293,7 +368,8 @@
             "success",
             "no-bid",
             "error",
-            "rejected"
+            "rejected-gdpr",
```

Comment:
Will the status ever be `rejected-gdpr`? I don't see this string used anywhere else in the code

Reply:
Not yet, but that is to come in another phase.

I can remove it though